### PR TITLE
Paper: PyBMRB: Data visualization tool for BioMagResBank

### PR DESCRIPTION
**Pull request by kumar-physics**
I apologize for the delay. I know I missed the deadline. This method of writing paper in a GitHub is new to me and it took time to understand the workflow.  I request you to accept my paper. 

Thanks,
Kumaran 

*Editor*: @mepa

*Reviewers*: @tkoyama010

### CODE REVIEW

**Comment by scoobies** [comment]
Hi!, I'm @scoobies, a friendly bot.

Type ```@scoobies help``` to discover how I can help you.

**Comment by kumar-physics** [comment]
@scoobies help

I am not sure whether I followed the workflow correctly to submit the paper. May I know whether my paper is considered for review. Once again I apologize for late submission. Thanks

**Comment by deniederhut** [comment]
@scoobies assign @mepa as editor

**Comment by scoobies** [comment]
Assigned! @mepa is now the editor

**Comment by mepa** [comment]
> I am not sure whether I followed the workflow correctly to submit the paper. May I know whether my paper is considered for review. Once again I apologize for late submission. Thanks

Hi @kumar-physics, thanks for your submission. I see the successfully compiled paper on the [build server page](https://procbuild.scipy.org/), so all looks well. Please continue to proofread and make any planned updates as soon as possible since reviewers, once assigned, may have a bit less time for reviewing due to the late submission.

**Comment by kumar-physics** [comment]
Thanks for your feedback. I have corrected few more typo and committed my first revision

**Comment by deniederhut** [comment]
@scoobies add @tkoyama010 to reviewers

**Comment by scoobies** [comment]
@tkoyama010 added to the reviewers list!

**Comment by deniederhut** [comment]
We've been having difficulty finding reviewers for some papers, so to make sure everyone has enough time to do their reviews, we will be bumping the deadline for the initial review back to June 17th.

**Comment by tkoyama010** [comment]
Thanks for your invitation. I will review as soon as possible.

File: papers/kumaran_baskaran/00_bibderwalt.rst [hunk 1]
```diff
@@ -0,0 +1,137 @@
...
+
+.. figure:: Figure1.jpg
+   
+   BMRB data processing workflow. :label:`workflow`
+
+
+
+In response to the demise of DEVise, BMRB developed graphic libraries in Python (PyBMRB)  that utilize more modern interactive visualization tools, such as the Plotly visualization tool kit, to reproduce the most commonly used features of DEVise with interactive visualizations. PyBMRB features single-entry (peak position simulation for NMR spectrum) and database-wide visualizations (histograms).
```
Comment (tkoyama010) [inline papers/kumaran_baskaran/00_bibderwalt.rst]: We need to [cite Plotly](https://plotly.com/chart-studio-help/citations/).

File: papers/kumaran_baskaran/00_bibderwalt.rst [hunk 2]
```diff
@@ -0,0 +1,137 @@
...
..., open archive for data generated by Nuclear Magnetic Resonance (NMR) spectroscopy of biological systems. NMR spectroscopy is unique among biophysical approaches in its ability to provide a broad range of atomic and higher-level information relevant to the structural, dynamic, and chemical properties of biological macromolecules, as well as report on metabolite and natural product concentrations in complex mixtures and their chemical structures. NMR-STAR is the official data format of BMRB and BMRB provides python parser (PyNMRSTAR https://github.com/uwbmrb/PyNMRSTAR), a data visualization tool (PyBMRB https://github.com/uwbmrb/PyBMRB) and an Application Program Interface (API)(BMRB-API https://github.com/uwbmrb/BMRB-API) to access the BMRB archive. PyBMRB displays the chemical shifts data in each entry as a simulated NMR spectrum and to generates database-wide chemical shift histograms of different atom types in proteins and nucleic acids. PyBMRB provides access to BMRB data through the API and generates portable and interactive visualizations as a single html file. It also supports  data visualization workflows using Jupyter Notebooks, which can be both easily created and shared.
```
Comment (tkoyama010) [inline papers/kumaran_baskaran/00_bibderwalt.rst:36]: Can we cite Jupyterlab which is the newer version of Jupyter Notebook? (I could not find the citation of it)

**Review by tkoyama010 — COMMENTED** [review]
@kumar-physics Awesome paper! I checked the contents and following.
- [x] no typos
- [x] no missing references

Please resolve the comment.

**Review by tkoyama010 — APPROVED** [review]

**Comment by tkoyama010** [comment]
@kumar-physics  Thanks for changes. LGTM.

**Comment by deniederhut** [comment]
@scoobies mark needs more review

**Comment by deniederhut** [comment]
@scoobies check references

**Comment by scoobies** [comment]
Checking the BibTeX entries failed with the following error: 
```
Couldn't find bibliography entry
```

File: publisher/build_paper.py [hunk 3]
```diff
@@ -49,6 +49,7 @@ def rst2tex(in_path, out_path):
 
     base_dir = os.path.dirname(__file__)
     out_file = shutil.copy(status_file, out_path)
+    out_file = os.path.join(out_path,'draft.sty')
```
Comment (deniederhut) [inline publisher/build_paper.py]: Please do not modify files outside of your paper directory

**Review by deniederhut — CHANGES_REQUESTED** [review]

File: publisher/build_paper.py (hunk 3 above)
Comment (kumar-physics) [reply publisher/build_paper.py]: Sorry. It wasn't working on my system, so I tweaked a bit.

Comment (kumar-physics) [reply publisher/build_paper.py]: Should I do anything to fix this.. or its already fixed?

Comment (deniederhut) [reply publisher/build_paper.py]: I'd like you to undo this change. If authors modify the build system it causes giant merge conflicts when we try to publish.

**Comment by kumar-physics** [comment]
Done.  build_paper.py restored to its original

File: publisher/build_paper.py (hunk 3 above)
Comment (kumar-physics) [reply publisher/build_paper.py]: Done. build_paper.py restored.

Comment (deniederhut) [reply publisher/build_paper.py]: Thanks!

**Review by deniederhut — APPROVED** [review]

**Comment by mepa** [comment]
@scoobies mark ready

File: papers/kumaran_baskaran/mybib.bib [hunk 4]
```diff
@@ -0,0 +1,122 @@
...
+@inproceedings{soton403913,
+       booktitle = {Positioning and Power in Academic Publishing: Players, Agents and Agendas},
+          editor = {Fernando Loizides and Birgit Scmidt},
+           title = {Jupyter Notebooks - a publishing format for reproducible computational workflows},
+          author = {Thomas Kluyver and Benjamin Ragan-Kelley and Fernando P{\'e}rez and Brian Granger and Matthias Bussonnier and Jonathan Frederic and Kyle Kelley and Jessica Hamrick and Jason Grout and Sylvain Corlay and Paul Ivanov and Dami{\'a}n Avila and Safia Abdalla and Carol Willing and  Jupyter development team},
+       publisher = {IOS Press},
+            year = {2016},
+           pages = {87--90},
+             url = {https://eprints.soton.ac.uk/403913/},
+        abstract = {It is increasingly necessary for researchers in all fields to write computer code, and in order to reproduce research results, it is important that this code is published. We present Jupyter notebooks, a document format for publishing code, results and explanations in a form that is both readable and executable. We discuss various tools and use cases for notebook documents.}
```
Comment (mepa) [inline papers/kumaran_baskaran/mybib.bib:87]: Please add
```
doi = {10.3233/978-1-61499-649-1-87}
```

Comment (kumar-physics) [reply papers/kumaran_baskaran/mybib.bib:87]: added

**Pull request merged**